lib/dockly/docker.rb

```diff
@@ -33,12 +33,27 @@ def build_env(hash = nil)
   end
 
   def generate!
-    image = generate_build
+    image = find_image_by_repotag || generate_build
     export_image(image)
   ensure
     cleanup([image]) if cleanup_images
   end
 
+  def export_only
+    if image = find_image_by_repotag
+      info "Found image by repo:tag: #{repo}:#{tag} - #{image.inspect}"
+      export_image(image)
+    else
+      raise "Could not find image"
+    end
+  end
+
+  def find_image_by_repotag
```

Comment:
So, with the change I just made, this is only used with exporting, not when packing a docker in the deb.  Now, when running `dockly:docker:prepare:x`, it'll build the image, and the export will only ever export.  I think this is better.  `dockly:docker:prepare:x` will always force the rebuild.